src/integration-test/charles-client.ts

```diff
@@ -193,6 +193,20 @@ export default class CharlesClient {
    * NOTIFICATION
    */
 
+  public async getTeamNotificationConfigurations(teamId: number): Promise<JsonApiEntity[]> {
+    const response = await this.fetchJson<ResponseMulti>(
+      `/api/teams/${teamId}/relationships/notification`,
+    );
+    return response.data;
+  }
+
+  public async getProjectNotificationConfigurations(projectId: number): Promise<JsonApiEntity[]> {
```

Comment:
A little surprisingly, the endpoint for the project-scoped notification configurations already existed, meaning there are no upcoming PRs.